app/services/csv/sales_log_csv_service.rb

```diff
@@ -161,6 +169,21 @@ def sales_log_attributes
       @user.support? ? final_attributes : final_attributes - SUPPORT_ONLY_ATTRIBUTES
     end
 
+    def sales_log_definitions
+      definitions = @user.variable_definitions.sales
+
+      definitions.group_by { |record| [record.variable, record.definition] }
+                 .map do |_, options|
+        exact_match = options.find { |definition| definition.year == @year && definition.user_type == @user.user_type }
+        next exact_match if exact_match
+
```

Comment:
Did we end up checking if the definitions are/should be different across different user types?

Reply:
We will check this. In any case though, should there be two different definitions one for 'user' and one for 'support' user_type the best match will be taken, if it exists, if not it should default to the best match which will the 'user' definition if there is no 'support' version. Should there be two and later a decision is made to just have one, all that needs to be done is to delete the 'support' record and support users will get the 'user' definition.

Reply:
Yeah, that makes sense, I'm a bit concerned tho that we're over-engineering to accommodate use case that might not happen. And if we're optimising for the best definition for a specific user, how would we extend this if we decide to have different definitions for providers vs coordinators?

Reply:
For sure, without these duplicate definitions the code would be simpler and cleaner. As it is though the source files include for the same attribute different definitions based on user type. Should that change and be confirmed not needed, we can adjust this.